src/popup/router/components/TabBar.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="tab-bar">
+    <router-link to="/account" :class="{ selected: $route.path === '/account' }">
```

Comment:
@etharner you have rebased an old version of this PR, don't revert already done fixes